### PR TITLE
add mms-python-adapter recipe

### PR DESCRIPTION
**Pull request by dlamoris**
Checklist
- [ x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml" 
- [ x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [ x] Source is from official source
- [ x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [ x] Build number is 0
- [ x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [ x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/mms-python-adapter/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  run:
+    - mms-python-client <4
+
+test:
+  imports:
+    - mms_python_adapter
+
+about:
+  home: http://openmbee.org
+  license: Apache 2.0
```
Comment (marcelotrevisani) [inline recipes/mms-python-adapter/meta.yaml]:
```suggestion
  license: Apache-2.0
```

File: recipes/mms-python-adapter/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,40 @@
+{% set name = "mms-python-adapter" %}
+{% set version = "1.2.5" %}
+{% set file_ext = "tar.gz" %}
+{% set hash_type = "sha256" %}
```
Comment (marcelotrevisani) [inline recipes/mms-python-adapter/meta.yaml]: I think you can remove a lot of those jinja variables and stay just with name and version

File: recipes/mms-python-adapter/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+{% set hash_value = "dded9d59246be03a8fb1f299b1b70b2dbf85dcea87076316e65d23f7bbc7fa7d" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ file_ext }}
+  '{{ hash_type }}': '{{ hash_value }}'
```
Comment (marcelotrevisani) [inline recipes/mms-python-adapter/meta.yaml]:
```suggestion
  sha256: dded9d59246be03a8fb1f299b1b70b2dbf85dcea87076316e65d23f7bbc7fa7d
```

File: recipes/mms-python-adapter/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+{% set file_ext = "tar.gz" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "dded9d59246be03a8fb1f299b1b70b2dbf85dcea87076316e65d23f7bbc7fa7d" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ file_ext }}
```
Comment (marcelotrevisani) [inline recipes/mms-python-adapter/meta.yaml]:
```suggestion
  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
```

File: recipes/mms-python-adapter/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+about:
+  home: http://openmbee.org
+  license: Apache-2.0
+  license_family: Apache
+  license_file: LICENSE
+  summary: QOL wrapper for mms-python-client
+
+extra:
+  recipe-maintainers: 
+    - HuiJun
```
Comment (scopatz) [inline recipes/mms-python-adapter/meta.yaml]: @HuiJun are you OK with being a maintainer?

Comment (HuiJun) [reply recipes/mms-python-adapter/meta.yaml]: LGTM

File: recipes/mms-python-adapter/meta.yaml [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - mms-python-client <4
```
Comment (scopatz) [inline recipes/mms-python-adapter/meta.yaml]: Does this not depend on Python at run time?

Comment (dlamoris) [reply recipes/mms-python-adapter/meta.yaml]: added python

**Comment by scopatz** [comment]
Thanks!

**Pull request merged**